src/react-apps/applications/ux-editor/src/components/FormComponent.tsx

```diff
@@ -103,11 +103,11 @@ const FormComponent = (props: IFormElementProps) => {
    * Render label
    */
   const renderLabel = (): JSX.Element => {
-    if (props.component.component === 'Header' ||
-      props.component.component === 'Paragraph' ||
-      props.component.component === 'Submit' ||
-      props.component.component === 'ThirdParty' ||
-      props.component.component === 'AddressComponent') {
+    if (props.component.type === 'Header' ||
```

Comment:
Are we making changes to the ui-editor internal state? Not just the new form layout?